darts/models/forecasting/pl_forecasting_module.py

```diff
@@ -425,6 +414,26 @@ def epochs_trained(self):
 
         return current_epoch
 
+    @staticmethod
+    def configure_torch_metrics(
```

Comment:
couldn't we instead add the loss and metrics to `PLForecastingModule.on_save_checkpoint/on_load_checkpoint` so that the saving / loading is automatically handled?

Reply:
Since they were excluded from `self.save_hyperparameters()` in `PLForecastingModule.__init__()`, I assumed that we wanted to avoid having them in the checkpoints but it should probably work. I will test and see if it works.

Reply:
Cool! I hope we will never discover why it was ignored in the first place...

I left the helper method `configure_torch_metrics()` but let me know if I should remove it since it's called only at one place in the code.